pkg/env/env.go

```diff
@@ -132,9 +132,10 @@ func (e *testEnv) Finish(funcs ...Func) types.Environment {
 func (e *testEnv) Run(ctx context.Context, m *testing.M) int {
 	setups := e.GetSetupActions()
 	// fail fast on setup, upon err exit
+	var err error
 	for _, setup := range setups {
-		// context not surfaced further
-		if _, err := setup.run(ctx); err != nil {
+		// context passed down to each setup
+		if ctx, err = setup.run(ctx); err != nil {
```

Comment:
Is this returned context being passed into Assess somewhere?

Reply:
@andrewsykim I realize what the issue is now.
No, no way to pass anything into running tests using testing.M
So, let me tinker with this to see if there is a better way (other than global context shared).